requirements-dev.txt

```diff
@@ -1,10 +1,14 @@
+bowler
 flake8
+flake8-black
 flake8-isort
 flake8-tidy-imports
 mypy
+sphinx
```

Comment:
Isn't this just for building the docs? It's already included in `docs/requirements.txt` so shouldn't be needed here.